e2e/vpc/vpc_create_volume_10iops_tier.go

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Cloud Container Service, 5737-D43
+ * (C) Copyright IBM Corp. 2018 All Rights Reserved.
+ * The source code for this program is not  published or otherwise divested of
+ * its trade secrets, irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+
+package vpc
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.uber.org/zap"
+
+	"github.com/IBM/ibmcloud-storage-volume-lib/lib/provider"
+)
+
+var _ = Describe("ibmcloud-storage-volume-lib", func() {
+	It("VPC: Create and delete VPC volume[profile: 10iops-tier]", func() {
+		volName := volumeName + "10iops"
+		volSize := volumeSize
+		Iops := iops
```

Comment:
Can we avoid setting Iops for `tiered` profiles as that does not require to set the iops, applicable for other profiles as well

Reply:
It profiles name.

```
    {
       "first":{
          "href":"https://us-south.iaas.cloud.ibm.com/v1/volume/profiles?limit=50"
       },
       "limit":50,
       "total_count":4,
       "profiles":[
          {
             "name":"10iops-tier",
             "family":"tiered",
             "generation":"gc",
             "href":"https://us-south.iaas.cloud.ibm.com/v1/volume/profiles/10iops-tier"
          },
          {
             "name":"5iops-tier",
             "family":"tiered",
             "generation":"gc",
             "href":"https://us-south.iaas.cloud.ibm.com/v1/volume/profiles/5iops-tier"
          },
          {
             "name":"custom",
             "family":"custom",
             "generation":"gc",
             "href":"https://us-south.iaas.cloud.ibm.com/v1/volume/profiles/custom"
          },
          {
             "name":"general-purpose",
             "family":"tiered",
             "generation":"gc",
             "href":"https://us-south.iaas.cloud.ibm.com/v1/volume/profiles/general-purpose"
          }
       ]
    }
```